mmv1/templates/terraform/examples/vertex_ai_model_upload_predict_schemata.tf.erb

```diff
@@ -3,9 +3,9 @@ resource "google_vertex_ai_models" "<%= ctx[:primary_resource_id] %>" {
   description  = "basic upload model"
 
   predict_schemata {
-    instance_schema_uri   = "gs://cloud-samples-data/vertex-ai/google-cloud-aiplatform-ci-artifacts/models/iris_xgboost/" # value needed
-    parameters_schema_uri = "gs://cloud-samples-data/vertex-ai/google-cloud-aiplatform-ci-artifacts/models/iris_xgboost/" # value needed
-    prediction_schema_uri = "gs://cloud-samples-data/vertex-ai/google-cloud-aiplatform-ci-artifacts/models/iris_xgboost/" # value needed
+    instance_schema_uri   = "gs://cloud-ai-platform-d357fffa-aab0-409b-8e4e-3af03de82d76/instance_schemas/job-5547038670390820864/analysis"
+    parameters_schema_uri = "gs://cloud-ai-platform-d357fffa-aab0-409b-8e4e-3af03de82d76/instance_schemas/job-5547038670390820864/analysis"
+    prediction_schema_uri = "gs://cloud-ai-platform-d357fffa-aab0-409b-8e4e-3af03de82d76/instance_schemas/job-5547038670390820864/analysis"
```

Comment:
We can't use this URI in the test- could you please update the test so that:
- Any files are stored in a test fixture folder, similar to [this](https://github.com/hashicorp/terraform-provider-google-beta/blob/28a5ecd7450d1b64505f48b0b3976e242f2d4b23/google-beta/services/compute/test-fixtures/openapi.yaml)
- Update the test to include provisioning a GCS bucket and object like [this](https://github.com/hashicorp/terraform-provider-google-beta/blob/ba69d710a93a09243293e88dca7bf7a12adff1a3/google-beta/services/appengine/resource_app_engine_standard_app_version_test.go#L129-L139)
- Set these fields in the ai model resource using attributes from the google_storage_bucket_object resource

Reply:
attempted these, but it still fails due to the yaml file not being in the right format:
```
Error: Error creating Models: googleapi: 
Error 400: List of found errors:     
1.Field: model.predict_schemata.prediction_schema_uri; 
Message: Invalid YAML file for prediction schema.
```

We'll need some feedback from google service team to help with the exact format that vertex-ai wants. it's not clear from the docs that I've come across.